composer.json

```diff
@@ -22,6 +22,9 @@
         "drupal/jsonapi": "1.0.0",
         "drupal/simple_oauth": "2.0-rc2"
     },
+    "require-dev": {
+        "roave/security-advisories": "dev-master"
```

Comment:
I would not add it by default. Because it slows down the install process. Composer has to download the package information for all packages in the Roave/SecurityAdvisories repo. I thinks its better to do it in a separate CI job.